### PR TITLE
FIX: console script invokation fails, add tests to catch it

### PR DESCRIPTION
**Pull request by gaborbernat**
as pointed out by @jurko-gospodnetic at https://github.com/tox-dev/tox/pull/711/files/2f63a4b8cbdbb36af0fec503d73adae658ad19d4#r157385227 thanks!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def main():
         author='holger krekel',
         author_email='holger@merlinux.eu',
         packages=['tox'],
-        entry_points={'console_scripts': 'tox=tox:cmdline\ntox-quickstart=tox._quickstart:main'},
+        entry_points={'console_scripts': ['tox=tox:cmdline',
+                                          'tox-quickstart=tox._quickstart:main']},
```
Comment (jurko-gospodnetic) [inline setup.py]: cool, I was wondering why this was not done this way originally, and assumed it was some sort of a backward compatibility hack

File: tests/test_z_cmdline.py [hunk 2]
```diff
@@ -865,3 +867,19 @@ def test_missing_env_fails(initproj, cmd):
...
+    assert result == 0
+
+
+def test_tox_quickstart_script():
+    result = subprocess.check_call(['tox-quickstart', '--help'])
+    assert result == 0
+
+
+def test_tox_cmdline(monkeypatch):
+    monkeypatch.setattr(sys, 'argv', ['caller_scrip', '--help'])
```
Comment (jurko-gospodnetic) [inline tests/test_z_cmdline.py]: what-looks-like-a-typo in the script name intentional?

**Comment by gaborbernat** [comment]
It's a typo but the name does not matter here 😁

File: .travis.yml [hunk 3]
```diff
@@ -28,10 +28,9 @@ matrix:
       language: generic
 
 before_install:
+ - pyenv versions
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source .travis-osx; fi
- # work around https://github.com/travis-ci/travis-ci/issues/8363
- - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then pyenv global system 3.5; fi
-
+
```
Comment (jurko-gospodnetic) [inline .travis.yml]: This line breaks the CI lint check - fixed in https://github.com/tox-dev/tox/pull/718.

Comment (gaborbernat) [reply .travis.yml]: thanks!

**Pull request merged**